src/SourceBrowser.SolutionRetriever/GitHubBasicAuthenticationProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+namespace SourceBrowser.SolutionRetriever
+{
+    internal static class GitHubBasicAuthenticationProvider
+    {
+        internal static string Token
+        {
+            get
+            {
+                // Note: production code contains a valid authentication token.
```

Comment:
The config needs to reside within `SourceBrowser.Site` project. I can't retrieve the value from a .config file in the `SourceBrowser.SolutionRetriever` project. Is that fine?
If yes, I will rename `LowPrivilegeUser.config` to `Security.config` or something else that better encompasses various sensitive values.